code/modules/shuttle/mobile_port/shuttle_move_callbacks.dm

```diff
@@ -115,6 +115,8 @@ All ShuttleMove procs go here
 	if(rotation)
 		shuttleRotate(rotation)
 
+	update_parallax_contents()
```

Comment:
On second thought a comment specifying why this call is needed would've been neat, my bad.